ksqldb-engine/src/test/java/io/confluent/ksql/engine/QueryPlanTest.java

```diff
@@ -62,6 +63,7 @@ public void shouldImplementEquals() {
         .addEqualityGroup(new QueryPlan(sources2, sink1, plan1, id1))
         .addEqualityGroup(new QueryPlan(sources1, sink2, plan1, id1))
         .addEqualityGroup(new QueryPlan(sources1, sink1, plan2, id1))
-        .addEqualityGroup(new QueryPlan(sources1, sink1, plan1, id2));
+        .addEqualityGroup(new QueryPlan(sources1, sink1, plan1, id2))
+        .testEquals();
```

Comment:
derp... did this test never do anything? 😂

Reply:
Nope, a whole set of these were missing this rather important call.  Only found two classes that had bad impls.